scenes/test_assert.txt

```diff
@@ -18,17 +18,23 @@
 *comment :::::: UTILITY routines ::::::
 *label test_start
 *params p_test_name
+*temp test_result true
 *set test_count + 1
 *set {log_buffer} &"TEST ${p_test_name}: "
 *return
 
+*label test_assert
+*params p_test_condition p_test_message
```

Comment:
I prefer an interface like test_finish, with expected and actual
the message can be generated using them